### ISSUE TITLE
No account creation, when submissions are closed. 

### ISSUE THREAD

**Issue by BCCNB** [Debilski/application-server#5]
Combined with a prominent display of the message that no applications are currently received on the home page.

**Comment by Debilski** [comment]
Yes. Additionally, it should be possible to send out invitation letters for a) candidates which have been granted extended deadlines and b) for reviewers.
Reviewers should never need to sign in through the normal user registration but only via invitation. (After clicking the invitation link, one should have to choose a password.)
